spec/features/partner_categories_page_spec.rb

```diff
@@ -86,6 +86,9 @@
           )
         end
 
+        # Have a "Become a Partner" button.
+        expect(page).to have_css("a[href='#{localize_url('become-a-partner', locale_id: locale[:id])}']", text: 'Become a Partner')
```

Comment:
so interesting that there's a space between `to` and `have_css` - different syntax I guess. Good to see how this is done in ruby though!

Reply:
This is:
```ruby
expect(page).to(have_css('…'))
```

Just in Ruby, unless you do a few specific things or chain off of it, the `()` are optional.